Assets/Scripts/Skill/IceSkill.cs

```diff
@@ -40,10 +40,12 @@ IEnumerator Attack(Vector3 position, float time)
         for( int i = 0; i < 5; ++i )
         {
             TakeDamage(position, Damage);
+            Vibration.VibrateAndroid(100);
```

Comment:
```suggestion
            Vibration.VibrateAndroid(30);
```
`Vibration.VibrateAndroid(100)` 호출 시 **100ms(=0.1초)** 동안 진동이 울립니다.
메테오, 회복 스킬과 동일한 진동이므로 도트 대미지의 진동을 짧게 하는 편이 좋을 수도 있습니다.